scanpipe/pipes/d2d.py

```diff
@@ -614,12 +614,23 @@ def _map_about_file_resource(project, about_file_resource, to_resources):
         # Cannot map anything without the about_resource value.
         return
 
+    ignored_resources = None
+    if extra_data := package_data.get("extra_data"):
+        ignored_resources = extra_data.get("ignored_resources")
+
     # Fetch all resources that are covered by the .ABOUT file.
     codebase_resources = to_resources.filter(path__contains=f"/{filename.lstrip('/')}")
     if not codebase_resources:
         # If there's nothing to map on the ``to/`` do not create the package.
         return
 
+    # Ignore resources for paths in `ignored_resources` attribute
+    if ignored_resources:
+        for resource_path in ignored_resources:
+            codebase_resources = codebase_resources.exclude(
+                path__contains=f"/{resource_path.lstrip('/')}"
+            )
```

Comment:
A better pattern would be to craft the list of lookups and call the queryset only once.

Reply:
This would be much better, implemented this, thanks!